check_synology.py

```diff
@@ -167,7 +167,7 @@ def exitCode():
 
     if warning and 1 == int(update_status_nr):
         state = 'WARNING'
-    if critical and [4|5] == int(update_status_nr):
+    if critical and int(update_status_nr) in [4, 5]:
```

Comment:
Thinking about this again, it would mean the `CRITICAL` status would be pulled on both `Disconnected` and `Others`. Do you think this is the right thing to do?

Let me consider another proposal:
- OK: Unavailable
- WARNING: Available
- UNKNOWN: Connecting, Disconnected, Others (all other states)

In this way, there would be no `CRITICAL` state at all. What do you think about it?

/cc @tonkenfo

Reply:
Sounds reasonable to me! 🚀

Reply:
I've implemented this change with 52d3b2f. In this case, because `-w` and `-c` do not transport any specific threshold values, I believe it is more convenient to report about the `state` in any case.

Also, a typo has been spotted and fixed with 249e11a.